### ISSUE TITLE
REALSR training code

### ISSUE THREAD

**Issue by ShiinaMitsuki** [Zheng222/IMDN#15]
Is there any plan for the release of the training code on REALSR dataset using IMDN_AS arch?

Thanks.

**Comment by Zheng222** [comment]
@ShiinaMitsuki The training processing of IMDN_AS is very similar with IMDN. If you are interested in IMDN_AS, please modify `train_IMDN.py` for training.

**Comment by ShiinaMitsuki** [comment]
> @ShiinaMitsuki The training processing of IMDN_AS is very similar with IMDN. If you are interested in IMDN_AS, please modify `train_IMDN.py` for training.

Thanks anyway. I modified one indeed, I trained on REALSR x4 part using the IMDN_x8 model(downsample the input by a factor of 8 to speed up), and I got PSNR ~29, SSIM ~0.82 on the REALSR validation set.

I wonder the performance being acceptable?

**Comment by Zheng222** [comment]
@ShiinaMitsuki The downsampling scale factors of fthe RealSR dataset is range from x2 to x4. Thus, I don't recommend x8 downsampling for our model. If you want to further improve the performance, you can train longer time when the learning rate is 2e-4 and 1e-4.
